rules/windows/impact_backup_file_deletion.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2021/10/01"
+maturity = "production"
+updated_date = "2021/10/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the deletion of Backup Files. Adversaries may delete Backup files
+to ensure that recovery from a Ransomware attack is less likely.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Backup Files Deletion"
+references = ["https://www.advintel.io/post/backup-removal-solutions-from-conti-ransomware-with-love"]
+risk_score = 47
+rule_id = "11ea6bec-ebde-4d71-a8e9-784948f8e3e9"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Impact"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.type == "deletion" and
+  (
+  /* Veeam Related Backup Files */
+  (file.extension : ("VBK", "VIB", "VBM") and
+  not process.executable : ("?:\\Windows\\Veeam\\Backup\\*", "?:\\Program Files\\Veeam\\Backup and Replication\\*")) or
+
+  /* Veritas Backup Exec Related Backup File */
+  (file.extension : ("BKF") and
+  not process.executable : ("?:\\Program Files\\Veritas\\Backup Exec\\*"))
```

Comment:
`file.extension == "BKF"` (no need for parentheses) , maybe also add the x86 equivalent path  `?:\\Program Files (x86)\\Veritas\\*` ,  `"?:\\Program Files (x86)\\Veeam\\*"`.

Reply:
Done, thanks!